### ISSUE TITLE
new block commenting stuff is broken

### ISSUE THREAD

**Issue by kingsleyh** [mint-lang/mint#91]
Hi

In latest master branch - when I have a comment like this I get a compiler error:

```
fun render : Html {
    <div class="pure-g">
      <div class="pure-u-1-1 in-center">
        <p>
          <{ "Perform Workout" }>
        </p>

        <br/>

       /*
       <div class="in-center">
         <h3>
            <{ "How hard did the skipping feel?" }>
          </h3>

         <Slider onChange={updateStoredEffort}/>

          <br/>
          <br/>

          <button
            onClick={(e : Html.Event) : Void => { recordCalibrationTime }}
            class="button-secondary button-xlarge">

            <{ "Done" }>

          </button>
        </div>
        */
      </div>
    </div>
  }
```

causes this compile error:

```
⚙ Ensuring dependencies... 220μs
Unhandled exception: Compiler not implemented for node #<Mint::Ast::Comment:0x110deaf60>! (Exception)
  from Mint::Compiler#compile<Mint::Ast::Node+>:NoReturn
  from Mint::Compiler#compile<Array(Mint::Ast::Node+)>:Array(String)
  from Mint::Compiler#compile<Mint::Ast::HtmlElement>:String
  from Mint::Compiler#compile<Array(Mint::Ast::Node+)>:Array(String)
  from Mint::Compiler#compile<Mint::Ast::HtmlElement>:String
  from Mint::Compiler#compile<Mint::Ast::Function, String>:String
  from Mint::Compiler#compile<Mint::Ast::Component>:String
  from Mint::Compiler#compile:String
  from Mint::Cli::Start#parse_and_run:Nil
  from Mint::Cli#parse_and_run:Nil
  from main
⚙ Compiling...
```

**Comment by kingsleyh** [comment]
@wisn this looks like it was introduced with your block comment updates

**Comment by gdotdesign** [comment]
It's fixed in 0.3.0
